canonical-websites/www.ubuntu.com
# Multicloud operations takeover

Blocked on https://github.com/canonical-websites/www.ubuntu.com/pull/3996

## Done
Build the multicloud operations takeover and remove the AI webinar takeover

## QA
- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/](http://0.0.0.0:8001/)
- Run through the following [QA steps](https://github.com/canonical-webteam/practices/blob/master/workflow/qa-steps.md)
- Check that homepage takeover matches [design](https://github.com/ubuntudesign/www.ubuntu.com-design/issues/153#issuecomment-418085878)

## Issue / Card
Fixes https://github.com/canonical-websites/www.ubuntu.com/issues/3957
